packages/core/src/components/popover/popover.scss

```diff
@@ -0,0 +1,94 @@
+@use '../../helpers';
+@use '../../mixins';
+@use 'sass:map';
+
+@mixin Popover() {
+  @if not mixins.includes('Popover') {
+    @include _Popover();
+  }
+}
+
+@mixin _Popover() {
+  .ods-popover-anchor {
+    pointer-events: none;
+    position: absolute;
+    z-index: 9999999;
+  }
+
+  .ods-popover {
+    height: fit-content;
+    max-width: helpers.space(50);
+    position: absolute;
+    width: max-content;
+
+    // position ==========>
```

Comment:
Is this a common convention for CSS comments?

Reply:
Nope, that's me. We could wrap it in a mixin too I suppose, or use a different comment style.
Happy to hear suggestions if you have them.
I did not spend a lot of time coming up with this, I just wanted to convey where it begins and where it ends.